tests/unit/algo/instance_segmentation/heads/test_custom_rtmdet_ins_head.py

```diff
@@ -71,7 +71,7 @@ def test_mask_pred(self, mocker) -> None:
         mask_head._bbox_mask_post_process(
             results[0],
             mask_feat,
-            cfg=None,
+            cfg=test_cfg,
         )
```

Comment:
Why the same lines (L65-69) are repeated again?

Reply:
hmm, that's a fair point. Maybe it was accidentally added during the merge of releases/2.0.0.